File: logica-programacao/javascript/005.js
/* 
Exercício 2: Você tem 50 balas e quer dividir igualmente entre 4 amigos.
Quantas balas cada amigo recebe?
*/

let quantidadeBalas = 50;
let quantidadeAmigos = 4;

let quantidadesRecebidas = Math.floor(quantidadeBalas / quantidadeAmigos);

let sobra = quantidadeBalas % quantidadeAmigos;

console.log(`A quantidade de balas é: ${quantidadeBalas}`);
console.log(`A quantiadade de amigos é: ${quantidadeAmigos}`);
console.log(`Cada um recebeu: ${quantidadesRecebidas}`);
console.log(`Sobraram ${sobra} no pote`);

File: logica-programacao/python/005.py
"""
Exercício 2: Você tem 50 balas e quer dividir igualmente entre 4 amigos.
Quantas balas cada amigo recebe?

"""

quantidade_balas = 50
quantidade_amigos = 4

quantidade_recebida = quantidade_balas // quantidade_amigos
sobra = quantidade_balas % quantidade_amigos

print(f"Quantidade de balas a serem divididas: {quantidade_balas}")
print(f"Quantidade de amigos: {quantidade_amigos}")
print(f"Cada um vai receber: {quantidade_recebida}")
print(f"Sobraram {sobra} balas no pote")